sklearn/linear_model/_least_angle.py

```diff
@@ -1582,6 +1582,9 @@ class LassoLarsCV(LarsCV):
     n_iter_ : array-like or int
         the number of iterations run by Lars with the optimal alpha.
 
+    active_ : list, length = n_alphas | list of n_targets such lists
```

Comment:
A list is also considered as an `array-like` object, alternatively you could specify the type of the element list (see the [guidelines for documentation](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation)).